package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "browser-extension",
+    "version": "1.0.0",
+    "main": "index.js",
+    "repository": "https://github.com/dylanirlbeck/browser-extension.git",
+    "author": "dylanirlbeck <dylanirlbeck@gmail.com>",
```

Comment:
Just realized that we should probably adjust this. I think the right course of action here is to use both [author](https://flaviocopes.com/package-json/#author) and [contributors](https://flaviocopes.com/package-json/#contributors) properties. We can specify the Art Institute of Chicago as the `author` and individuals as `contributors`. I think it's best to reserve the `contributors` list for people who are not employed by our organization.